ts/src/conductor.ts

```diff
@@ -324,26 +324,32 @@ export class Conductor {
     ]);
     runConductorProcess.stdin.write(LAIR_PASSWORD);
     runConductorProcess.stdin.end();
+    this.conductorProcess = runConductorProcess;
 
-    const startPromise = new Promise<void>((resolve) => {
+    // Wait for conductor startup to complete and admin ws to be available
+    let adminPortLogged = false;
+    const adminPortPromise = new Promise<string>((resolve) => {
       runConductorProcess.stdout.on("data", (data: Buffer) => {
         logger.info(data.toString());
-        const conductorLaunched = data.toString().match(/Conductor ready\./);
-        if (conductorLaunched) {
-          // This is the last output of the startup process.
+
+        if(!adminPortLogged) {
```

Comment:
This was causing some tests to flake as well -- the ADMIN_PORT magic string wasn't being read from stdout.